opengovsg/plumber
# Migrate to OGP DS's Masthead

## Description
We no longer need to maintain our own masthead since we can use chakra now.

## Tests
_Scrappy since Plumber still in PMF stage_

* Check that masthead renders properly ([closed](https://github.com/opengovsg/plumber/assets/135598754/16ca0993-89e7-446f-9797-91d576617e4d) + [expanded](https://github.com/opengovsg/plumber/assets/135598754/1cc27df5-dd1a-4615-a69e-718c04a0c246)) on both public and logged in surfaces.
* Check that flow creation, editing and deletion can still be done.
